spec.bs

```diff
@@ -3247,6 +3250,69 @@ a [=list=] of [=interest groups=] |bidIgs|, and a [=reporting context map=]
 
 </div>
 
+<div algorithm>
+To <dfn>handle server response private aggregation fields</dfn> given a [=server auction response=]
+|response|, a [=reporting context=] |reportingContext|, and a [=reporting bid key=] |reportingId|:
+
+1. [=Assert=] that these steps are running [=in parallel=].
+1. [=Commit server response private aggregation contributions=] given |response|'s
+  [=server auction response/component win private aggregation contributions=], |reportingContext|,
+  |reportingId| and false.
+1. [=Commit server response private aggregation contributions=] given |response|'s
+  [=server auction response/server filtered private aggregation reserved contributions=],
+  |reportingContext|, |reportingId| and true.
+1. [=Commit server response private aggregation contributions=] given |response|'s
+  [=server auction response/server filtered private aggregation non reserved contributions=],
+  |reportingContext|, |reportingId| and true.
+
+</div>
+
+<div algorithm>
+To <dfn>commit server response private aggregation contributions</dfn> given a
+[=server auction response=] |contributionsMap|, a [=reporting context=] |reportingContext|, a
+[=reporting bid key=] |reportingId| and a [=boolean=] |serverFiltered|:
+
+1. [=map/For each=] (|reportingOrigin|, |coordinator|, |event|) → |contributions| of |contributionsMap|:
+  1. Let |eventToContributionsMap| be a new [=Private Aggregation contributions=].
+  1. Let |scopingDetails| be a new [=scoping details=] with the [=struct/items=]:
+    : <a spec="private-aggregation-api" for="scoping details">get batching scope steps</a>
+    :: An algorithm that performs the following steps:
+        1. If |coordinator| is null, set |coordinator| to the [=default aggregation coordinator=].
+        1. Return the result of running [=get or create a batching scope=] given |reportingOrigin|,
+          |coordinator| and |reportingContext|.
+    : <a spec="private-aggregation-api" for="scoping details">get debug scope steps</a>
+    :: An algorithm that returns a new [=debug scope=].

```

Comment:
oh I see what you mean. Yeah I copied it so that it can be consistent with the other place, but not feel strongly. Directly setting the two scopes does look more readable, so updated.